packages/fast-components-react-base/src/slider/slider.tsx

```diff
@@ -628,22 +634,35 @@ class Slider extends Foundation<SliderHandledProps, SliderUnhandledProps, Slider
     };
 
     /**
-     *  Updates the direction
+     *  updates the direction in state if necessary
      */
     private updateDirection = (): void => {
+        const newDirection: Direction = this.getDirection();
+        if (newDirection !== this.state.direction) {
+            this.suspendActiveOperations();
+            this.setState({
+                direction: newDirection,
+            });
+        }
+    };
+
+    /**
+     *  gets the current direction
+     */
+    private getDirection = (): Direction | null => {
         if (this.rootElement.current === null) {
-            return;
+            return Direction.ltr;
         }
 
         const closest: Element = this.rootElement.current.closest(`[dir]`);
 
         if (closest === null) {
-            this.direction = Direction.ltr;
-            return;
+            return Direction.ltr;
         }
 
-        this.direction =
-            closest.getAttribute("dir") === "rtl" ? Direction.rtl : Direction.ltr;
+        return closest.getAttribute(Slider.dirPropName) === "rtl"
```

Comment:
I'd assume this can just be `this.dirPropName` to refer to the local private instance. Also "rtl" should be `Direction.ltr`.

Reply:
dirPropName is static so it doesn't exist on "this".